_includes/footer.html

```diff
@@ -47,24 +45,20 @@ <h6 class="post-title"><a href="blog-single.html">Lorem ipsm dolor sumit.</a></h
         <h3>Contact US</h3>
         <div class="widget">
           <div class="newsletter-wrapper">
-            <form method="post" id="subscribe-form" name="subscribe-form" class="validate">
-              <div class="form-group is-empty">
-                <input type="email" value="" name="Email" class="form-control" id="EMAIL" placeholder="Your email"
-                  required="">
-                <button type="submit" name="subscribe" id="subscribes" class="btn btn-common sub-btn"><i class="lni-pointer"></i></button>
-                <div class="clearfix"></div>
-              </div>
-            </form>
+            <ul>
+              <li><a href="mailto:{{ site.email }}">{{ site.email }}</a></li>
+            </ul>
           </div>
         </div>
 
         <div class="widget">
           <h5 class="widget-title">FOLLOW US ON</h5>
           <ul class="footer-social">
-            <li><a class="facebook" href="#"><i class="lni-facebook-filled"></i></a></li>
-            <li><a class="twitter" href="#"><i class="lni-twitter-filled"></i></a></li>
-            <li><a class="linkedin" href="#"><i class="lni-linkedin-filled"></i></a></li>
-            <li><a class="google-plus" href="#"><i class="lni-google-plus"></i></a></li>
+            <li><a class="github" href={{ site.github }}><i class="lni-github"></i></a></li>
+            <li><a class="facebook" href={{ site.facebook }}><i class="lni-facebook-filled"></i></a></li>
+            <li><a class="twitter" href={{ site.twitter }}><i class="lni-twitter-filled"></i></a></li>
+            <li><a class="instagram" href={{ site.instagram }}><i class="lni-instagram-filled"></i></a></li>
+            <li><a class="google-plus" href={{ site.google_plus }}><i class="lni-google-plus"></i></a></li>
```

Comment:
I think we should remove google-plus.

Change the order to the same of our website, please.